dspace/config/modules/rest.cfg

```diff
@@ -31,6 +31,7 @@ rest.projection.specificLevel.maxEmbed = 5
 # configuration properties.
 rest.properties.exposed = plugin.named.org.dspace.curate.CurationTask
 rest.properties.exposed = google.analytics.key
+rest.properties.exposed = websvc.opensearch.enable
```

Comment:
I believe you also will need to add `webui.feed.enable` to this list.  Keep in mind, RSS feeds are enabled when `webui.feed.enable=true`.  However, it looks like you've reworked them to depend more on OpenSearch...so, we may want to discuss whether having both `websvc.opensearch.enable` and `webui.feed.enable` makes sense or not.

Reply:
So the entire functionality is solely based on if `websvc.opensearch.enable` is set to true if so also on the angular side it enables so `webui.feed.enabled` is not needed at all